mac/build.sh

```diff
@@ -334,95 +161,64 @@ updatePlist() {
 
-    builder_heading "Building Keyman.app"
-    cd "$KM4MIM_BASE_PATH"
-    if $DO_PODS ; then
-        pod update
-        pod install
-    fi
+  builder_heading "Building Keyman.app"
 
-    cd "$KEYMAN_MAC_BASE_PATH"
-    execBuildCommand $IM_NAME "xcodebuild -workspace \"$KMIM_WORKSPACE_PATH\" $CODESIGNING_SUPPRESSION $BUILD_OPTIONS $BUILD_ACTIONS -scheme Keyman SYMROOT=\"$KM4MIM_BASE_PATH/build\""
-    if [ "$TEST_ACTION" == "test" ]; then
-        if [ "$CONFIG" == "Debug" ]; then
-            execBuildCommand "$IM_NAME tests" "xcodebuild $TEST_ACTION -workspace \"$KMIM_WORKSPACE_PATH\" $CODESIGNING_SUPPRESSION $BUILD_OPTIONS -scheme Keyman SYMROOT=\"$KM4MIM_BASE_PATH/build\""
-        fi
-    fi
-    updatePlist "$KM4MIM_BASE_PATH/build/$CONFIG/Keyman.app/Contents/Info.plist" "Keyman"
+  execBuildCommand $IM_NAME "xcodebuild -workspace \"$KMIM_WORKSPACE_PATH\" $CODESIGNING_SUPPRESSION $BUILD_OPTIONS $BUILD_ACTIONS -scheme Keyman SYMROOT=\"$KM4MIM_BASE_PATH/build\""
+  updatePlist "$KM4MIM_BASE_PATH/build/$CONFIG/Keyman.app/Contents/Info.plist" "Keyman"
 
-    if [ "$CONFIG" == "Debug" ]; then
-        ENTITLEMENTS_FILE=Keyman.Debug.entitlements
-    else
-        ENTITLEMENTS_FILE=Keyman.entitlements
-    fi
+  if builder_is_debug_build; then
+    ENTITLEMENTS_FILE=Keyman.Debug.entitlements
+  else
+    ENTITLEMENTS_FILE=Keyman.entitlements
 
     # We need to re-sign the app after updating the plist file
-    if $DO_CODESIGN ; then
-        execCodeSign eval --force --sign $CERTIFICATE_ID --timestamp --verbose --preserve-metadata=identifier,entitlements "$KM4MIM_BASE_PATH/build/$CONFIG/Keyman.app/Contents/Frameworks/Sentry.framework"
+    execCodeSign eval --force --sign $CERTIFICATE_ID --timestamp --verbose --preserve-metadata=identifier,entitlements "$KM4MIM_BASE_PATH/build/$CONFIG/Keyman.app/Contents/Frameworks/Sentry.framework"
 
-        execCodeSign eval --force --sign $CERTIFICATE_ID --timestamp --verbose --preserve-metadata=identifier,entitlements "$KM4MIM_BASE_PATH/build/$CONFIG/Keyman.app/Contents/Frameworks/KeymanEngine4Mac.framework"
+    execCodeSign eval --force --sign $CERTIFICATE_ID --timestamp --verbose --preserve-metadata=identifier,entitlements "$KM4MIM_BASE_PATH/build/$CONFIG/Keyman.app/Contents/Frameworks/KeymanEngine4Mac.framework"
 
-        execCodeSign eval --force --sign $CERTIFICATE_ID --timestamp --verbose -o runtime \
-            --entitlements "$KM4MIM_BASE_PATH/$ENTITLEMENTS_FILE" \
-            --requirements "'=designated => anchor apple generic and identifier \"\$self.identifier\" and ((cert leaf[field.1.2.840.113635.100.6.1.9] exists) or ( certificate 1[field.1.2.840.113635.100.6.2.6] exists and certificate leaf[field.1.2.840.113635.100.6.1.13] exists and certificate leaf[subject.OU] = \"$DEVELOPMENT_TEAM\" ))'" \
-            "$KM4MIM_BASE_PATH/build/$CONFIG/Keyman.app"
-    fi
+    execCodeSign eval --force --sign $CERTIFICATE_ID --timestamp --verbose -o runtime \
```

Comment:
I've gone ahead and eliminated the calls that are straightforward from my perspective, but these are a few evals that I'm less clear on how to fix at the moment.  I'm away from my Mac dev machine this week due to travel, and I'd definitely want to test changes to these lines locally first.

Reply:
`git blame` points me to https://github.com/keymanapp/keyman/pull/10243.  There's no linked issue and the description is very sparse, so I'm currently unable to infer details about why the `eval` vs `direct` pattern is in place.  It's a fairly recent PR as well, so I get the feeling I should leave the `eval` bits here unless directed otherwise by someone with "headspace documentation"..

Reply:
Yeah, I am not sure why `eval` is needed. I would have to experiment, and I have a vague recollection of this being more difficult than it seemed like it should have to be. Let's leave the execCodeSign eval calls for now -- it works after all.